src/metadatas.rs

```diff
@@ -15,9 +15,14 @@ pub struct Metadata {
     pub issues_count: std::collections::BTreeMap<IssueType, usize>,
     pub has_fares: bool,
     pub has_shapes: bool,
+    // some stops have a pickup_type or drop_off_type equal to "ArrangeByPhone"
+    pub some_stops_need_phone_agency: bool,
+    // some stops have a pickup_type or drop_off_type equal to "CoordinateWithDriver"
+    pub some_stops_need_phone_driver: bool,
 }
 
 pub fn extract_metadata(gtfs: &gtfs_structures::RawGtfs) -> Metadata {
+    use gtfs_structures::PickupDropOffType::*;
```

Comment:
I know it's done below, but I don't like glob import, I think it's better to just
`use gtfs_structures::PickupDropOffType` and then `PickupDropOffType::ArrangeByPhone`

Reply:
ok j'avais effectivement copié sur l'autre ligne ;)